themes/gatsby-theme-blog/gatsby-config.js

```diff
@@ -95,12 +75,7 @@ module.exports = {
       },
     },
     {
-      resolve: `gatsby-plugin-google-analytics`,
-      options: {
-        // trackingId: `ADD YOUR TRACKING ID HERE`,
-      },
-    },
-    {
+      // should this be included in the theme?
       resolve: `gatsby-plugin-manifest`,
```

Comment:
Does having the manifest plugin make sense for a theme? I could see how it could make sense, but seems like the type of thing that might be better handled at the site level and in the starter that uses this theme

Reply:
i vote site level.

Reply:
I vote theme. Every site needs it and it's easy to drive from the theme by asking for a logo image and using e.g. the primary color. It's rarely something people touch after installing.

Reply:
I can revert this change. One thing I'm wondering about is how multiple plugins like this would be merged, assuming the notes theme also has the manifest plugin

Reply:
Hmm yeah, we'd need the same prop merging capabilities.

Reply:
So yeah, I guess keeping this stuff at the site level does start to make sense 👍

Reply:
Changed my mind haha 😂 I'm with y'all now!